Egg Timer/ttViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
```

Comment:
why we need this file? if it's not required, please delete

Reply:
I have deleted the file.